src/PerfView.Tests/StackViewer/StackWindowTests.cs

```diff
@@ -32,54 +32,54 @@ public StackWindowTests(ITestOutputHelper testOutputHelper)
         {
         }
 
-        [WpfFact]
```

Comment:
Sorry about this, must have been some temp change when testing this and it slipped through because the file showed up as collapsed in the GH PR. reverted to the original